.github/workflows/post-merge-to-build.yml

```diff
@@ -47,4 +47,4 @@ jobs:
           container-sign-kms-key-arn: ${{ secrets.BUILD_CONTAINER_SIGN_KMS_KEY }}
           template-file: deploy/template.yaml
           role-to-assume-arn: ${{ secrets.GH_ACTIONS_ROLE_ARN }}
-          ecr-repo-name: ${{ secrets.ECR_REPOSITORY }}
+          ecr-repo-name: ${{ secrets.BUILD_ECR_REPOSITORY }}
```

Comment:
why not just change the value?

Reply:
I did but just think the explicit names better, we have a dev_ecr_repository secret, so having a build one makes more sense to me

Reply:
Resolving - discussed the name change BUILD_ECR_REPOSITORY for clarity and agreed.
Ticket being raised for the other F.E's to align with this.